content/docs/forwarding-refs.md

```diff
@@ -4,51 +4,51 @@ title: Forwarding Refs
 permalink: docs/forwarding-refs.html
 ---
 
-Ref forwarding is a technique for automatically passing a [ref](/docs/refs-and-the-dom.html) through a component to one of its children. This is typically not necessary for most components in the application. However, it can be useful for some kinds of components, especially in reusable component libraries. The most common scenarios are described below.
+*Ref forwarding* adalah sebuah teknik untuk meneruskan [ref](/docs/refs-and-the-dom.html) secara otomatis melalui komponen ke salah satu anaknya. Ini biasanya tidak diperlukan untuk sebagian besar komponen dalam aplikasi. Namun, ini bisa berguna untuk beberapa jenis komponen, terutama di pustaka komponen yang dapat digunakan kembali. Skenario paling umum dijelaskan di bawah ini.
 
-## Forwarding refs to DOM components {#forwarding-refs-to-dom-components}
+## *Forwarding refs* ke komponen DOM {#forwarding-refs-to-dom-components}
```

Comment:
```suggestion
## Mengoper refs ke komponen DOM {#forwarding-refs-to-dom-components}
```

- Don't use italics in headings.
- I think we can translate this to "mengoper *refs*".

Reply:
Noted, I will update it soon.
So, for every 'Forwarding refs' should be translated to 'mengoper refs' or not?

Reply:
Oh, one more thing, should we use "mengoper" or "meneruskan"? Because there are a lot of "forwarding" words, and some translated as "mengoper" but some translated as "meneruskan". 🤔